astropy/wcs/wcs.py

```diff
@@ -84,7 +84,7 @@
            'NoWcsKeywordsFoundError', 'InvalidTabularParametersError']
 
 
-if six.PY3 or platform.system() == 'Windows':
+if (not six.PY2) or platform.system() == 'Windows':
```

Comment:
I don't think brackets affect performance, so I am okay with leaving them in for clarity.